File: C_Programs/PrintGradeDescription.cpp
/*Write a program in C to accept a grade and declare the equivalent description : 
Grade 	Description
E 	Excellent
V 	Very Good
G 	Good
A 	Average
F 	Fail*/

#include<stdio.h>
#include<conio.h>

int main()
{
	char ch;
	printf("\nEnter the Grade from E, V, G, A, F : ");
	scanf("%c",&ch);
	
	if(ch=='E')
		printf("\nGrade %c : Excellent",ch);
	if(ch=='V')
		printf("\nGrade %c : Very Good",ch);
	if(ch=='G')
		printf("\nGrade %c : Good",ch);
	if(ch=='A')
		printf("\nGrade %c : Average",ch);
	if(ch=='F')
		printf("\nGrade %c : Fail",ch);					
	return 0;
}

File: Basic_Java_Programs/PrintGradeDescription.java
/*Write a program to accept a grade and declare the equivalent description :
Grade 	Description
E 	Excellent
V 	Very Good
G 	Good
A 	Average
F 	Fail*/
import java.util.*;

public class PrintGradeDescription {
    public static void main(String[] args)
    {
        char ch;
        Scanner sc=new Scanner(System.in);

        System.out.println("Enter the Grade from E, V, G, A, F : ");
        ch=sc.next().charAt(0);

        if(ch=='E')
            System.out.println("Grade "+ch+" : Excellent");
        if(ch=='V')
            System.out.println("Grade "+ch+" : Very Good");
        if(ch=='G')
            System.out.println("Grade "+ch+" : Good");
        if(ch=='A')
            System.out.println("Grade "+ch+" : Average");
        if(ch=='F')
            System.out.println("Grade "+ch+" : Fail");
    }
}